laos_gggi/distance_to_rivers.py

```diff
@@ -0,0 +1,12 @@
+import pandas as pd
```

Comment:
Put all this in `statistics.py`, it doesn't make sense to have a file for one function

Reply:
Perfect.